excel.go

```diff
@@ -447,3 +450,42 @@ func updateMACFinUsers(f *excelize.File, input *Input, client S3ClientAPI, env E
 
 	return nil
 }
+
+// password-protect excel file
+func protectExcelWorkbook(filename string) (string, error) {
+	dir := filepath.Dir(filename)
+	outFilename := filepath.Join(dir, "protected-"+filepath.Base(filename))
+	cat := exec.Command("cat", filename)
```

Comment:
We may want to verify if what we are executing is found in the path (cat and secure-spreadsheet) since the complete path isn't being used. There is an easy way to do this with `exec.LookPath()`. [Doc](https://pkg.go.dev/os/exec#LookPath).

Reply:
Good point. `exec` does this. Better for me to check and fail gracefully if the command is not found.